src/analyses/call_graph_helpers.cpp

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************\
+
+Module: Function Call Graph Helpers
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Function Call Graph Helpers
+
+#include "call_graph_helpers.h"
+
+static std::set<irep_idt> get_neighbours(
```

Comment:
Nit: document (along with the rest of the functions in here). 

~Also, controversial suggestion, but `neighbours` doesn't mean much in the context of a callgraph, presumably it is the functions called by the specified function?~ Done using function successor and predecessor

Reply:
I suppose actually it is either the called functions, or the callsites, depending on the value of `forwards` - this should definitely be documented.

Reply:
Ah I see docs are in the header file. Personally prefer this, though I think we've been putting them in the source file - don't know if a decision has been reached @peterschrammel ?